docs/docs/launch-manual-v2/sections/ppr-frontend/index.md

```diff
@@ -0,0 +1,21 @@
+---
+lang: en-US
+title: Pulsar Package Registry Frontend Website
+description: Using and Navigating the Pulsar Package Registry Website
+---
+
+# Pulsar Package Registry Frontend Website
+
+The Package Registry Frontend Website viewable on [https://web.pulsar-edit.dev](https://web.pulsar-edit.dev/) is the frontend counterpart to the Pulsar Package Registry. Allowing an easy way for anyone on any platform to view and browse the packages that are available to install on Pulsar.
```

Comment:
I suppose that was thought as a way to help people understand simply how it works since I do remember some initial questions about if the search was handled differently from the website, api, and pulsar.

But yeah suppose your point is true. That's all any website is. So I agree we can remove it.